roles/k8s_addons/tasks/helm.yml

```diff
@@ -1,27 +1,8 @@
 
-# - name: Download helm
```

Comment:
Latest helm (3) (needs upgrade) doesn't need a permanent installation in kubernetes anymore. :)